batch/batch/driver/instance_pool.py

```diff
@@ -253,11 +246,11 @@ async def create_instance(self, cores=None, max_idle_time_msecs=None):
             if machine_name not in self.name_instance:
                 break
 
-        if self.live_total_cores_mcpu // 1000 < int(USCENTRAL1_CORES * 0.9):
+        if self.live_total_cores_mcpu // 1000 < 4_000:
             zones = ['us-central1-a', 'us-central1-b', 'us-central1-c', 'us-central1-f']
             zone = random.choice(zones)
         else:
-            zone = random.choices(ZONES, ZONE_WEIGHTS)[0]
+            zone = random.choices(self.zones, self.zone_weights)[0]
```

Comment:
Did you test this code path works since you're only using it with a big cluster?

Reply:
I lived deployed it to the large cluster, it is working admirably.